packages/core/src/thing-description.ts

```diff
@@ -193,6 +193,9 @@ export interface AllOfSecurityScheme extends SecurityScheme {
 }
 export type ComboSecurityScheme = AllOfSecurityScheme | OneOfSecurityScheme;
 
+export interface AutoSecurityScheme extends SecurityScheme, TDT.AutoSecurityScheme {
```

Comment:
VERY minor: I would have put `AutoSecurityScheme` right after `NoSecurityScheme` so that the order matches with `SecurityType`.

Since `SecurityType` gets removed anyway I REALLY don't mind